src/coord/src/catalog/builtin.rs

```diff
@@ -1574,6 +1576,15 @@ mod tests {
                         "funcs with oid {} don't match names: {} in mz, {} in pg",
                         imp.oid, func.name, pg_fn.name
                     );
+
+                    // Complain, but don't fail, if argument oids don't match.
+                    // TODO: make these match.
+                    if imp.arg_oids != pg_fn.arg_oids {
+                        println!(
```

Comment:
Printing this to users' terminals wherever we disagree with PG seems a little odd to me; I think this will nag the engineering team more than actual users, which is good, but I wonder if there's any alternatives that let us hide our dirty laundry to users?

Reply:
This is in an ignored test function that devs have to opt in to running.